src/main/java/hudson/plugins/copyartifact/monitor/LegacyJobConfigMigrationMonitor.java

```diff
@@ -126,7 +125,7 @@ public boolean isActivated() {
     /**
      * @return data holding the list of jobs to warn.
      */
-    @VisibleForTesting LegacyMonitorData getData() {
+    /* Visible for testing */ LegacyMonitorData getData() {
```

Comment:
`@Restricted(NoExternalUse.class)` is appropriate. I'll add a commit for this to this pull request later.